src/OAuth1/Server.php

```diff
@@ -71,7 +71,7 @@ public function getTokenCredentials(TemporaryCredentials $temporaryCredentials,
                 $response = $client->post($uri, $headers, $bodyParameters)->send();
             }
         } catch (BadResponseException $e) {
-            return $this->handleTokenCredentialsBadResponse($e);
+            $this->handleTokenCredentialsBadResponse($e);
```

Comment:
this seems like a functional change?

Reply:
It does not, `handleTokenCredentialsBadResponse` does not return anything (`void`), it only throws an exception.